test/onnx/test_pytorch_common.py

```diff
@@ -51,12 +51,12 @@ def wrapper(*args, **kwargs):
 def skipIfUnsupportedMinOpsetVersion(min_opset_version):
     def skip_dec(func):
         @functools.wraps(func)
-        def wrapper(self, *args, **kwargs):
-            if self.opset_version < min_opset_version:
+        def wrapper(test_class, *args, **kwargs):
```

Comment:
Why the name change?
Is the first argument really a class? I thought it was an instance of the TestCase, in which case the old name seems better?

Reply:
I proposed the change initially. I agree after giving it another thought

Reply:
some convention uses `cls` too. `self` is clearer in a sense this is instance method.

Reply:
`test_case`? Since type(self) == TestCase